portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,28 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private final ArrayList<String> messages =  new ArrayList<>();
+  Gson gson = new Gson();
+
+  @Override
+  public void init() {
+    messages.add("It would only take one hour to drive to space.");
+    messages.add("Google's first office was a rented garage.");
+    messages.add("Walmart has lower acceptance rate than Harvard.");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Salvador!</h1>");
+    String json = convertToJsonUsingGson(messages);
+
+    response.setContentType("application/json");
+    response.setCharacterEncoding("UTF-8");
+    response.getWriter().print(json);
```

Comment:
get rid of the method convertToJsonUsingGson and the variable `json`. Just change this line to
`response.getWriter().print(gson.toJson(messages);`